controllers/mattermost/mattermost/file_store.go

```diff
@@ -35,7 +41,56 @@ func (r *MattermostReconciler) checkExternalFileStore(mattermost *mmv1beta.Matte
 	return mattermostApp.NewExternalFileStoreInfo(mattermost, *secret)
 }
 
-func (r *MattermostReconciler) checkOperatorManagedMinio(mattermost *mmv1beta.Mattermost, reqLogger logr.Logger) (*mattermostApp.FileStoreInfo, error) {
+func (r *MattermostReconciler) checkLocalFileStore(mattermost *mmv1beta.Mattermost, reqLogger logr.Logger) (mattermostApp.FileStoreConfig, error) {
+	// Default 1Gi volume using default storage class
```

Comment:
```suggestion
	// Default 50Gi volume using default storage class
```

Reply:
Unsure about the actual need for this comment, since we could just check the constant.

Reply:
Removed comment.